conf/messages

```diff
@@ -9,6 +9,32 @@ navTitle.nonAgent = Manage your VAT account
 navTitle.agent = Your client’s VAT details
 navTitle.makingTaxDigitalForVat = VAT
 
+month.1 = January
+month.2 = February
+month.3 = March
+month.4 = April
+month.5 = May
+month.6 = June
+month.7 = July
+month.8 = August
+month.9 = September
+month.10 = October
+month.11 = November
+month.12 = December
+
+month.short.1 = Jan
+month.short.2 = Feb
+month.short.3 = Mar
+month.short.4 = Apr
+month.short.5 = May
+month.short.6 = Jun
+month.short.7 = Jul
+month.short.8 = Aug
+month.short.9 = Sept
```

Comment:
See comment in dev channel from Paul regarding use of "Sep" instead of "Sept"

Reply:
Thats been pushed up